### PR TITLE
docs: update release guide

### PR DESCRIPTION
**Pull request by lucasmoura**
## Proposed Commit Message
docs: update release guide

Add an extra step on the RELEASE.md docs and fix the script we are using to upload the new release to past distros

## Test Steps
Reproduce the steps in the script and run the new script

## Desired commit type::

 - [ ] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [x] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist:

 - [ ] I have updated or added any unit tests accordingly
 - [ ] I have updated or added any integration tests accordingly
 - [x] I have updated or added any documentation accordingly

### CODE REVIEW

File: RELEASES.md [hunk 1]
```diff
@@ -37,45 +37,52 @@ Below is the procedure used to release ubuntu-advantage-client to an Ubuntu seri
  3. git fetch upstream
  4. git rebase --onto pkg/ubuntu/devel 27.0.2 27.1
  5. git checkout -B upload-27.1-impish # to create a new local branch name based on your detached branch in step 4
- 6. git push `<your_launchpad_user>` upload-27.1-impish
- 7. Create a PPA for upload reviewers at `https://launchpad.net/~<YOUR_LP_USER>/+activate-ppa`
- 8. Push to your test PPA so upload reviewers can install easily test packages easily during review
+ 6. Make sure the changelog version contains the release version on the name (For example, 27.1~21.10.1)
+ 7. git push `<your_launchpad_user>` upload-27.1-impish
+ 8. Create a PPA for upload reviewers at `https://launchpad.net/~<YOUR_LP_USER>/+activate-ppa`
+ 9. Push to your test PPA so upload reviewers can install easily test packages easily during review
```
Comment (blackboxsw) [inline RELEASES.md:43]: too many "easily"s :) not your fault
```suggestion
 9. Push to your test PPA so upload reviewers can easily install and test packages during review
```

**Review by blackboxsw — APPROVED** [review]
minor nit and +1

**Comment by lucasmoura** [comment]
@blackboxsw updated

**Review by blackboxsw — APPROVED** [review]
CI failure due to git timeout on simplestreams. +1

**Pull request merged**